lib/core/request.js

```diff
@@ -320,11 +320,13 @@ class Request {
 
   static [kHTTP2CopyHeaders] (raw) {
     const rawHeaders = raw.split('\r\n')
-
     const headers = {}
+
     for (const header of rawHeaders) {
       const [key, value] = header.split(': ')
 
+      if (value == null || value.length === 0) continue
```

Comment:
why is this needed?

Reply:
This is a small fix, basically if the content of the `headers` was an empty string, the object was initialized with `{ '': undefined }`

Reply:
I don't understand why the header would ever be an empty string